channeld/channeld_htlc.h

```diff
@@ -71,7 +71,6 @@ static inline struct htlc *htlc_get(struct htlc_map *htlcs, u64 id, enum side ow
 	return NULL;
 }
 
-/* FIXME: Move these out of the hash! */
```

Comment:
Maybe drop a short comment on what dead precisely means for safety-of-deletion?

Reply:
Maybe whoever named the function originally should write the comment. I almost deleted the function entirely since it's an implementation detail that doesn't belong in the public interface (i.e., the header file), but I left it there to minimize the diff.